stock_request/models/stock_request_abstract.py

```diff
@@ -98,6 +98,7 @@ def _compute_product_qty(self):
         string="Routes",
         compute="_compute_route_ids",
         readonly=True,
+        store=True,
```

Comment:
Can store True be avoidable in v14? It doesn't make sense to store this relations in the database...

Reply:
If it's I cannot create a new stock request or open the form view, it allows me to create a stock request from the stock request order, but If I want to create a new stock request it raises an error.

I don't know if it's a bug from Odoo 14.

The error is the following 

````

Error:
Odoo Server Error

Traceback (most recent call last):
  File "/Users/Ramitos/Github/14.0/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/Users/Ramitos/Github/14.0/odoo/odoo/http.py", line 682, in dispatch
    result = self._call_function(**self.params)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/http.py", line 358, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/http.py", line 346, in checked_call
    result = self.endpoint(*a, **kw)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/http.py", line 911, in __call__
    return self.method(*args, **kw)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/http.py", line 530, in response_wrap
    response = f(*args, **kw)
  File "/Users/Ramitos/Github/14.0/odoo/addons/web/controllers/main.py", line 1359, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/Users/Ramitos/Github/14.0/odoo/addons/web/controllers/main.py", line 1351, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/api.py", line 396, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/api.py", line 383, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/models.py", line 6177, in onchange
    result['value'] = snapshot1.diff(snapshot0, force=first_call)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/models.py", line 6036, in diff
    record[name]
  File "/Users/Ramitos/Github/14.0/odoo/odoo/models.py", line 5620, in __getitem__
    return self._fields[key].__get__(self, type(self))
  File "/Users/Ramitos/Github/14.0/odoo/odoo/fields.py", line 2407, in __get__
    return super().__get__(records, owner)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/fields.py", line 980, in __get__
    raise ValueError("Compute method failed to assign %s.%s" % (record, self.name))
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/Ramitos/Github/14.0/odoo/odoo/http.py", line 638, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/Users/Ramitos/Github/14.0/odoo/odoo/http.py", line 314, in _handle_exception
    raise exception.with_traceback(None) from new_cause
ValueError: Compute method failed to assign stock.request(<NewId 0x106ff7a90>,).route_ids
````

Reply:
Can you apply https://github.com/OCA/stock-logistics-warehouse/pull/983, this error is happening also en v13, but with the fix, it should not appear anymore.